File: project3/Team3/projectBQ/src/main/java/bitc/full502/projectbq/dto/MinStockDto.java
package bitc.full502.projectbq.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class MinStockDto {

    private long id;
    private long itemId;
    private String itemName;
    private String itemCode;
    private String itemManufacturer;
    private long itemPrice;
    private long categoryId;
    private String categoryName;
    private String categoryKrName;
    private long stockQuantity;
    private long standardQty;
    private long safetyQty;
    private String minStockStatus;
}

File: project3/Team3/App_BQ/app/src/main/java/bitc/full502/app_bq/data/model/MinStockDto.kt
package bitc.full502.app_bq.data.model

data class MinStockDto(
    val id: Long,
    val itemId: Long,
    val itemName: String,
    val itemCode: String,
    val itemManufacturer: String,
    val itemPrice: Long,
    val categoryId: Long,
    val categoryName: String,
    val categoryKrName: String,
    val stockQuantity: Long,
    val standardQty: Long,
    val safetyQty: Long,
    val minStockStatus: String
)